scripts/eurostat/health_determinants/common/denominator_mcf_generator.py

```diff
@@ -25,12 +25,13 @@
 from dcid_existence import check_dcid_existence
 # pylint: enable = wrong-import-position
 
-_MODULE_DIR = os.path.join(_COMMON_PATH, "..", "bmi")
+_MODULE_DIR = os.path.join(_COMMON_PATH, "..", "social_environment")
 
-_INPUT_MCF_FILE_PATH = os.path.join(_MODULE_DIR, "output_files",
-                                    "eurostat_population_bmi.mcf")
-_OUTPUT_MCF_FILE_PATH = os.path.join(_MODULE_DIR, "output_files",
-                                     "eurostat_population_bmi_deno.mcf")
+_INPUT_MCF_FILE_PATH = os.path.join(
+    _MODULE_DIR, "output_files", "eurostat_population_social_environment.mcf")
+_OUTPUT_MCF_FILE_PATH = os.path.join(
+    _MODULE_DIR, "output_files",
+    "eurostat_population_social_environment_deno.mcf")
```

Comment:
these should be created from flag values, so that the code can still run for the former imports.

please then update the README, or add a script that will be able to run this and the other health imports.

Reply:
reminder to please address this comment. we should still be able to run the code for `population_bmi` after this pr is merged.

Reply:
Created flag, so that code can run for other imports.